src/Microsoft.AspNetCore.Mvc.TagHelpers/ImageTagHelper.cs

```diff
@@ -121,18 +123,15 @@ public override void Process(TagHelperContext context, TagHelperOutput output)
                 // not function properly.
                 Src = output.Attributes[SrcAttributeName].Value as string;
 
-                output.Attributes.SetAttribute(SrcAttributeName, _fileVersionProvider.AddFileVersionToPath(Src));
+                output.Attributes.SetAttribute(SrcAttributeName, FileVersionProvider.AddFileVersionToPath(ViewContext.HttpContext.Request.PathBase, Src));
```

Comment:
If you're feeling inspired refactor the file version into a variable to make debugging easier. If not, good as is. Same true for other TagHelpers.

Reply:
meh